src/libcommon/utility/types.cpp

```diff
@@ -51,6 +51,8 @@ std::string toString(NodeType e) {
             return "File";
         case NodeType::Directory:
             return "Directory";
+        default:
```

Comment:
**Suggestion.** Add a unit test that raises the default clause.

Reply:
Done in commit [3221738](https://github.com/Infomaniak/desktop-kDrive/pull/304/commits/3221738991d4803921d7a885541deb5a3862d66e)